squash_bot/list_timetable/commands.py

```diff
@@ -0,0 +1,82 @@
+import typing
+from urllib.parse import urljoin
+
+import requests
+
+from squash_bot.core import command as _command
+from squash_bot.core import command_registry, lambda_function
+from squash_bot.core.data import dataclasses as core_dataclasses
+from squash_bot.settings import base as settings_base
+
+
+@command_registry.registry.register
+class ListTimetableCommand(_command.Command):
+    name = "list-timetable"
+    description = "List the timetable between two datetimes "
+    options = (
+        _command.CommandOption(
```

Comment:
Show next n days days rather than having to provide long iso formats

Reply:
Done